### ISSUE TITLE
Modal popup to display ingredient info

### ISSUE THREAD

**Issue by keenanmcnamara12** [keenanmcnamara12/WhatsForDinnerMVC#8]
Adding to the menu edit page.

If we can get this on the menu edit page, we should include this on the menu list page as well.

**Comment by keenanmcnamara12** [comment]
Initial thoughts - looks like we're hitting a page refresh at some point so the "shown modal" isn't really persisting.  Going to attempt to address by implementing an HttpPost that updates a public property in the menu class.  Similarly the closing action will have to clear the flag.  

Additional note - need to keep a close eye on whether the selected reciped will change to the correct row when the view button is clicked.  If so, we can load the modal without any additionally handling to make sure the selected recipe is update.  Otherwise we'll need to account for it. 

Also, also - only implemented for add recipe, should implement for the delete recipe as well.  Possibly on the menulist page as well (certainly could modularize some of the cshtml if this were a long term project to make this more maintainable).

**Comment by keenanmcnamara12** [comment]
There's definitely some competition going on between the page refresh because of the row selection and the modal change.  It seems like the page refresh was not necessary since I was able to successfully delete a menu after commenting out the refresh in the onSuccess for the selection change js fn.  

Also noticed that the add function is not working.  Had to correct the looping to use SearchRecipeResults instead of Recipes.  The fix must have been reverted in some of the merging.
